pkl-codegen-java/src/test/kotlin/org/pkl/codegen/java/JavaCodeGeneratorTest.kt

```diff
@@ -336,77 +331,72 @@ class JavaCodeGeneratorTest {
         |  @Deprecated
         |  public static final class DeprecatedClass {
       """
-        .trimMargin()
-    assertThat(javaCode).contains(expected)
+    assertContains(expected, javaCode)
   }
 
-  @Test
-  fun `deprecated module class with message`() {
-    for (generateJavadoc in listOf(false, true)) {
-      val javaCode =
-        generateJavaCode(
-          """
-          @Deprecated{ message = "module class deprecation message" }
-          module DeprecatedModule
-          
-          propertyInDeprecatedModuleClass : Int = 42
+  @ParameterizedTest
+  @ValueSource(booleans = [false, true])
+  fun `deprecated module class with message`(generateJavadoc: Boolean) {
+    val javaCode =
+      generateJavaCode(
         """
-            .trimIndent(),
-          generateJavadoc = generateJavadoc
-        )
-      val expectedJavadoc =
-        if (!generateJavadoc) ""
-        else
-          """
-          |/**
-          | * @deprecated module class deprecation message
-          | */
+      @Deprecated{ message = "module class deprecation message" }
+      module DeprecatedModule
+      
+      propertyInDeprecatedModuleClass : Int = 42
+    """,
```

Comment:
Oh Spotless, wherefore art though so spotty :(
```suggestion
          @Deprecated{ message = "module class deprecation message" }
          module DeprecatedModule

          propertyInDeprecatedModuleClass : Int = 42
        """,
```

[throughout]

Reply:
In my latest commit, I formatted the code exactly as it falls out of the IntelliJ editor.
ktfmt successfully destroyed this formatting, and ktfmt is the boss.